autogen

```diff
@@ -425,7 +425,7 @@ EOF
           echo "import ${BASE_PY/%.py/}"
         fi
         # Add basic bootstrap code.
-        cat <<EOF
+      cat <<EOF
```

Comment:
This was correct previously: the `cat <<EOF` should be indented here, since it's part of the higher-level `if` statement, and it will fit under the `# Add basic bootstrap code.` comment. Why remove the indent here?